templates/vue-ts/src/Feedback.vue

```diff
@@ -36,29 +32,37 @@ const { snapshot, send } = useMachine(feedbackMachine, {
       </button>
     </div>
 
-
-
     <div v-if="snapshot.matches('thanks')" class="step">
       <h2>Thanks for your feedback.</h2>
 
-      <p v-if="snapshot.context.feedback.length > 0">"{{ snapshot.context.feedback }}"</p>
-
+      <p v-if="snapshot.context.feedback.length > 0">
+        "{{ snapshot.context.feedback }}"
+      </p>
     </div>
 
-
-    <form v-if="snapshot.matches('form')" class="step" @submit.prevent="send({ type: 'submit' })">
+    <form
+      v-if="snapshot.matches('form')"
+      class="step"
+      @submit.prevent="send({ type: 'submit' })"
+    >
       <h2>What can we do better?</h2>
-      <textarea name="feedback" rows={4} placeholder="So many things..." @input="(ev) => {
-        send({
-          type: 'feedback.update',
-          value: (ev.target as HTMLTextAreaElement).value
-        })
-      }" />
+      <textarea
+        name="feedback"
+        rows="{4}"
```

Comment:
I don't understand why it's adding quotes around `{4}`. Also don't know Vue so I'm not sure if `rows={4}` is supposed to be valid syntax or not.

Reply:
let's revert that change if possible then and land this PR without it